update_mkdocs_yml.py

```diff
@@ -43,7 +43,7 @@
         ]
     }
 
-mkdocs["extra"]["repo_name"] = mkdocs["repo_url"].replace("https://github.com/", "")
+mkdocs["extra"]["repo_name"] = mkdocs["repo_url"].replace("https://github.com/", "").rstrip("/")
```

Comment:
Would this change suffice for removing trailing slashes from the repo_url? @froschdesign